### DISCUSSION TITLE
Web UI / Management interface

### DISCUSSION THREAD

**Discussion by outbackdingo** [xmidt-org/xmidt#77] [General]
I take it from the documents so far on the RDK site, there does not exists a management UI for xmidt to act as an interface to manage / view devices. Is this a correct assumption? and if so how are people actually managing devices ?

**Comment by Sachin4403** [comment]
Hello @outbackdingo 

We have an internal tool through which we are managing this. since everyone will have their own set of data model parameters due to which  everyone is having their own UI to manage the devices on top of Xmidt/WebPA.

**Reply by outbackdingo** [reply]
is there a published data model / api we can use to build a ui to manage this ???

**Reply by Sachin4403** [reply]
You can use Tr1d1um API to manage all  of these but you need to manage ACL in front of Tr1d1um API so that  every User in the UI will be able perform only some set of actions.

**Reply by outbackdingo** [reply]
I dont see where there is a published document for Tr1d1um API or im looking in the wrong places i checked the docs and github, any way i can get you in a one on one for 15-20 mins this would be much faster
